### PR TITLE
[parquet] add user transactions table to parquet

### PR DESCRIPTION
**Pull request by bowenyang007**
Modified schema based on feedback from https://github.com/aptos-labs/aptos-indexer-processors/pull/513

New schema: 
```
    pub txn_version: i64,
    pub block_height: i64,
    pub parent_signature_type: String,
    pub sender: String,
    pub sequence_number: i64,
    pub max_gas_amount: u64,
    pub expiration_timestamp_secs: u64,
    pub gas_unit_price: u64,
    pub gas_used: u64,
    #[allocative(skip)]
    pub timestamp: chrono::NaiveDateTime,
    pub entry_function_id_str: String,
    pub epoch: i64,
    pub gas_fee_payer_address: Option<String>,
    pub is_transaction_success: bool,
    pub storage_refund_amount: u64,
    pub num_signatures: i64,
```

### CODE REVIEW

File: rust/processor/src/db/common/models/user_transactions_models/parquet_user_transactions.rs [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    pub block_height: i64,
+    pub parent_signature_type: String,
+    pub sender: String,
+    pub sequence_number: i64,
+    pub max_gas_amount: u64,
+    pub expiration_timestamp_secs: u64,
+    pub gas_unit_price: u64,
+    pub gas_used: u64,
+    #[allocative(skip)]
+    pub timestamp: chrono::NaiveDateTime,
```
Comment (ying-w) [inline rust/processor/src/db/common/models/user_transactions_models/parquet_user_transactions.rs:39]: can you reorder + rename columns to be

1. txn_version
2. block_height
3. block_timestamp (from timestamp)
4. epoch
5. sender
6. sequence_number
7. entry_function_id_str
8. expiration_timestamp_secs
9. parent_signature_type
10. gas_fee_payer_address
12. gas_used
13. gas_unit_price
14. max_gas_amount
15. storage_refund_amount -> storage_refund_octa
16. is_transaction_success
17. num_signatures

(not sure what the skip thing means)

File: rust/processor/src/db/common/models/user_transactions_models/parquet_user_transactions.rs [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+                .map(Signature::get_signature_type)
+                .unwrap_or_default(),
+            sender: standardize_address(&user_request.sender),
+            sequence_number: user_request.sequence_number as i64,
+            max_gas_amount: user_request.max_gas_amount,
+            expiration_timestamp_secs: user_request
+                .expiration_timestamp_secs
+                .as_ref()
+                .expect("Expiration timestamp is not present in user txn")
+                .seconds as u64,
```
Comment (ying-w) [inline rust/processor/src/db/common/models/user_transactions_models/parquet_user_transactions.rs:102]: is u64 enough here?

File: rust/processor/src/db/common/models/user_transactions_models/parquet_user_transactions.rs (hunk 1 above)
Comment (bowenyang007) [reply rust/processor/src/db/common/models/user_transactions_models/parquet_user_transactions.rs:39]: skip means skip right @yuunlimm ?

File: rust/processor/src/db/common/models/user_transactions_models/parquet_user_transactions.rs (hunk 2 above)
Comment (bowenyang007) [reply rust/processor/src/db/common/models/user_transactions_models/parquet_user_transactions.rs:102]: yea should be.

Comment (bowenyang007) [reply rust/processor/src/db/common/models/user_transactions_models/parquet_user_transactions.rs:102]: b/c the source says u64 so it cannot exceed that.

File: rust/processor/src/db/common/models/user_transactions_models/parquet_user_transactions.rs (hunk 1 above)
Comment (yuunlimm) [reply rust/processor/src/db/common/models/user_transactions_models/parquet_user_transactions.rs:39]: yeah, we could use different way to measure the size, but i don't think it's a big deal

Comment (bowenyang007) [reply rust/processor/src/db/common/models/user_transactions_models/parquet_user_transactions.rs]: reordered.

Comment (ying-w) [reply rust/processor/src/db/common/models/user_transactions_models/parquet_user_transactions.rs]: hmm that's true, i redo the order in bigquery i think

one more rename: max_gas_amount -> max_gas_octa

Comment (ying-w) [reply rust/processor/src/db/common/models/user_transactions_models/parquet_user_transactions.rs]: also does `gas_fee_payer_address` need standardization? (faa has correct length)

Comment (bowenyang007) [reply rust/processor/src/db/common/models/user_transactions_models/parquet_user_transactions.rs]: address is already standardized.

**Review by yuunlimm — APPROVED** [review]

**Pull request merged**